src/graph/rg_matrix/rg_matrix.h

```diff
@@ -297,6 +297,11 @@ GrB_Info RG_Matrix_wait
 	bool force_sync
 );
 
+GrB_Type RG_Matrix_type
```

Comment:
Please follow the same signature as GraphBLAS
`GrB_Info GxB_Matrix_type(GrB_Type *type, RG_Matrix A)`